deps/wazuh_testing/wazuh_testing/scripts/simulate_agents.py

```diff
@@ -403,6 +406,28 @@ def main():
                             type=str, help='''File containing a list of packages to be sent by syscollector.
                             One package per line. Default is None.''', required=False, default=None,
                             dest='syscollector_packages_list_file')
+    
+    arg_parser.add_argument('--vulnerability-legacy-messages',
+                            help='Enable prior 4.2 agents syscollector format. Default is False.',
+                            required=False,
+                            action='store_true',
+                            default=False,
+                            dest='vulnerability_legacy_messages')
+    
+    arg_parser.add_argument('--vulnerability-batch-size',
+                            type=int,
+                            help='Number of vulnerabilities per agent',
+                            required=False,
+                            default=10,
+                            dest='vulnerability_batch_size')
+    
+    arg_parser.add_argument('--vulnerability-packages-list-file', 
+                            metavar='<vulnerability_packages_list_file>',
+                            type=str, 
+                            help='''File containing a list of packages to be sent by syscollector. One package per line. Default is None.''', 
```

Comment:
The default value isn't None, is `deps/wazuh_testing/wazuh_testing/data/vulnerability_parsed_packages.json`
Also, use single quotes instead.